### PR TITLE
ISSUE #2958 - Custom scrollbars in V5

### LINKED ISSUE #2958: Review the scrollbars in v5
### Description
We are showing the scrollbars all the time in v5. I'm not sure that's a good look and feel.

### Goals
- [ ] Have the scrollbars replaced by the react scrollbards.

### Tasks
- [ ] Get rid of having default scrollbars showing when is not necessary 
- [ ] Use simplebar for the content of the modals (fixed header and footer)
- [ ] Use simple bar for revisions  (take in consideration minimum and maximum height)
- [ ] Use simple bar  in federations and containers
- [ ] In containers the revisions sublist  should have a fix height after 5 items. 
- [ ] In user permissions lets have the react scrollbar in the whole page.

### Related Resources
[Scrollbars in modals](https://www.figma.com/file/djgxvRKOaMMyutswn5yywU/Teamspace-(live)?node-id=3160%3A51485)

### PR DESCRIPTION
**Pull request by The-Daniel**
This fixes #2958 

#### Description
- Added custom scrollbars for V5.
- Scrollable area control created as ScrollArea control.
- Scrollbars added to all modals, to all dashboard pages, and to revision details (when clicking on container in container dashboard)

#### Test cases
- Open all modals and observe that scroll works as intended.
- Open all dashboard pages and observe that scroll works as intended.
- View a container's revisions using a mixed number of revisions (The list should reach a max of five revisions before scrolling).

#### Notes
- For most cases the contents of the scroll area did not need to be modified much. The exception to this is the revision details.
- The revisionsDetails pages are a little bit chaotic. In the styling a lot of values depend on other values (i.e. they're sums of padding and margin). Could it be useful define variables at the top of the page like I've done with the list item height?
- I wasn't sure how large to make the max height of the modals. I ended up using 70% of the screen height, but this can change.

### CODE REVIEW

File: frontend/src/v5/ui/controls/modal/formModal/formDialog.styles.ts [hunk 1]
```diff
@@ -56,6 +56,7 @@ export const Title = styled(Typography).attrs({
 export const FormDialogContent = styled(DialogContent)`
 	margin: ${({ zeroMargin }) => (zeroMargin ? '0' : '20px 34px 43px')};
 	display: inline-flex;
+	width: fill-available;
```
Comment (Amantini1997) [inline frontend/src/v5/ui/controls/modal/formModal/formDialog.styles.ts:59]: I know about webkit-fill-available, but I have never seen this property before, are you sure it is supported?

**Review by Amantini1997 — CHANGES_REQUESTED** [review]

File: frontend/src/v5/ui/controls/modal/formModal/formDialog.styles.ts (hunk 1 above)
Comment (The-Daniel) [reply frontend/src/v5/ui/controls/modal/formModal/formDialog.styles.ts:59]: Ha, yeah. I was intending to go back to this but forgot.

**Comment by sanmont3drepo** [comment]
Seems good to me if its ok with @george-hadfield  I'd say is approved.

**Review by sanmont3drepo — APPROVED** [review]

**Review by Amantini1997 — APPROVED** [review]

**Pull request merged**